packages/design-system/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -30,13 +30,19 @@ export class ChoiceList extends React.PureComponent {
    * Creates a list of Choice components
    */
   choices() {
+    console.log(this.props.choices);
     return this.props.choices.map((choiceProps) => {
       choiceProps.inversed = this.props.inversed;
       choiceProps.name = this.props.name;
       choiceProps.onBlur = (this.props.onBlur || this.props.onComponentBlur) && this.handleBlur;
       choiceProps.onChange = this.props.onChange;
       choiceProps.size = this.props.size;
       choiceProps.type = this.props.type;
+      choiceProps.className = choiceProps.className
+        ? choiceProps.className.concat(' ', this.props.errorMessage ? 'ds-c-choice--error' : null)
+        : this.props.errorMessage
+        ? 'ds-c-choice--error'
+        : null;
```

Comment:
You want to be using the `classNames` dependency to handle this logic. Generally its not a good idea to use multiple tertiary statements like this, as it makes it hard to read.

I believe this section can be simply accomplished with:
```
choiceProps.className = classNames(choiceProps.className, {
   'ds-c-choice--error': this.props.errorMessage,
})
```